tests/schemas/test_openapi.py

```diff
@@ -130,6 +130,36 @@ class View(generics.GenericAPIView):
         assert responses['200']['content']['application/json']['schema']['required'] == ['text']
         assert list(responses['200']['content']['application/json']['schema']['properties'].keys()) == ['text']
 
+    def test_response_body_nested_serializer(self):
+        path = '/'
+        method = 'POST'
+
+        class NestedSerializer(serializers.Serializer):
+            number = serializers.IntegerField()
+
+        class Serializer(serializers.Serializer):
+            text = serializers.CharField()
+            nested = NestedSerializer()
+
+        class View(generics.GenericAPIView):
+            serializer_class = Serializer
+
+        view = create_view(
+            View,
+            method,
+            create_request(path),
+        )
+        inspector = AutoSchema()
+        inspector.view = view
+
+        responses = inspector._get_responses(path, method)
+        schema = responses['200']['content']['application/json']['schema']
+        assert schema['required'] == ['text', 'nested']
+        assert list(schema['properties'].keys()) == ['text', 'nested']
```

Comment:
Except we have an OrderedDict issue here. 

sort the list. (I don't want to pull in OrderedDict for this.)